libs/dm-localization/src/lib/loadTrans.ts

```diff
@@ -407,4 +407,188 @@ export default function loadTrans() {
       )
     })
   })
+
+  // Dump Somnia
+  const Somnia = {
```

Comment:
we will have to make some changes to how she loads sheets, possibly make a new sheet template function just for her? 
@nguyentvan7 can you take this work, since you had the most experience?

Reply:
You already did it, but I was wondering if we could just put the translated values in char_somnia.json, then load them here in loadTrans and redump them to char_somnjaGen.json. So we didn't have to have the one-off handling in the frontend code

Reply:
This would make a weird dependency of `dm-localization`-> `localization`, which doesnt quite work because `localization` runs in `code-space` and `dm-localization` runs in `/dist` space.